app/controllers/api/v1/stores_controller.rb

```diff
@@ -1,11 +1,57 @@
 class Api::V1::StoresController<ApplicationController 
     def index
         stores=Store.all
-        render json: stores
+        render json: store
     end
+    
     def create
-        store= Store.new(name:params[:name], category:params[:category], open_time:params[:open_time])
-        store.save
+        store = Store.new(name: params[:name], category: params[:category], open_time: params[:open_time], password: params[:password])
+        if　store.save
+            render json: store
+        else
+            render json {message: store.errors.full_messages.joins(",")}
+        end
+    end
+
+    def show
+        store = Store.find_by(id: params[:id])
+        render json: {message: "the store is not found"}, status:not_found if store.blank?
```

Comment:
```suggestion
        render json: { message: "the store is not found" }, status: not_found if store.blank?
```
ちょっと細かいけど、rubyでは上記の様に空白を入れる書き方が好まれるよ！